src/koans/26_transducers.clj

```diff
@@ -0,0 +1,23 @@
+(ns koans.26-transducers
+  (:require [koan-engine.core :refer :all]))
+
+(def xfms
```

Comment:
Can we change this to an easier-to-pronounce name, even if it's just "transformers" / "transforms" / whatever?

Reply:
yep, understand that's probably confusing to a beginner. Will change